### DISCUSSION TITLE
在windows server 2012 r2中显示异常

### DISCUSSION THREAD

**Discussion by 493505110** [HMBSbige/BilibiliLiveRecordDownLoader#94] [Q&A] [answered]
![Screenshot_20210609_055915_com.microsoft.rdc.android.jpg](https://i.loli.net/2021/06/09/N8aqEQJCu9PAepn.jpg)
建议添加命令行工具

**Comment by HMBSbige** [comment, marked as answer]
系统缺少[字体](https://aka.ms/SegoeFonts)
